java/jedis-compatibility/README.md

```diff
@@ -46,6 +46,7 @@ dependencies {
     <groupId>io.valkey</groupId>
     <artifactId>valkey-glide-jedis-compatibility</artifactId>
     <version>${valkey-glide.version}</version>
```

Comment:
Why do we use parameters here, but use actual versions and classifiers in the gradle example?